src/commands/kv/namespace/create.rs

```diff
@@ -14,19 +14,19 @@ pub fn run(
     user: &GlobalUser,
     binding: &str,
 ) -> Result<(), failure::Error> {
-    let target = manifest.get_target(env, is_preview)?;
-    kv::validate_target(&target)?;
+    let account_id = manifest.get_account_id(env)?;
+    let worker_name = manifest.worker_name(env);
     validate_binding(binding)?;
 
-    let mut title = format!("{}-{}", target.name, binding);
+    let mut title = format!("{}-{}", worker_name, binding);
     if is_preview {
         title.push_str("_preview");
     }
     let msg = format!("Creating namespace with title \"{}\"", title);
     message::working(&msg);
 
     let client = http::cf_v4_client(user)?;
-    let result = create(&client, &target, &title);
+    let result = create(&client, &account_id, &title);
```

Comment:
we should validate the presence of account id in this command; since manifest allows it to be missing or empty, we will get back the cryptic api gateway error.